pom.xml

```diff
@@ -223,7 +216,7 @@
         <dependency>
             <groupId>uk.gov.ons.ctp.common</groupId>
             <artifactId>test-framework</artifactId>
-            <version>10.49.10</version>
+            <version>10.49.11-SNAPSHOT</version>
```

Comment:
Still waiting for [https://travis-ci.org/ONSdigital/rm-common-test-framework/builds/406169220?utm_source=github_status&utm_medium=notification](this)

Reply:
https://github.com/ONSdigital/rm-common-test-framework/commit/5f67008b5758013f89fccaebc8958853b328cce0